### PR TITLE
[music] Listeners should not be able to hold a negative value

### PR DESCRIPTION
**Pull request by razzeee**
Not sure if this field is used anymore, but if it is, it shouldn't show up as a negative value.

(Yes, I finally got that annoyed by that negative value in my debugger)

### CODE REVIEW

File: xbmc/music/tags/MusicInfoTag.cpp [hunk 1]
```diff
@@ -470,7 +472,8 @@ void CMusicInfoTag::SetUserrating(char rating)
 
 void CMusicInfoTag::SetListeners(int listeners)
 {
- m_listeners = listeners;
+  listeners = std::max(listeners, 0);
+  m_listeners = listeners;
```
Comment (tamland) [inline xbmc/music/tags/MusicInfoTag.cpp]: Surely, one line and one assignment should be sufficient

**Comment by tamland** [comment]
Are you sure? If you do that you can't distinguish between 0 listeners, and listener value not being set

**Comment by Razzeee** [comment]
Your right. But I'm not sure if there is value for this kind of info. I could also change it to - 1.

File: xbmc/music/tags/MusicInfoTag.cpp (hunk 1 above)
Comment (Razzeee) [inline xbmc/music/tags/MusicInfoTag.cpp]: Updated

**Comment by Razzeee** [comment]
Any objections?

jenkins build this please

**Pull request merged**